### PR TITLE
Confirm password field is added

### PR DESCRIPTION
**Pull request by shraddha761**
Issue reference - #336 

Successfully added confirm password field

### CODE REVIEW

**Comment by shraddha761** [comment]
@narayan954 please have a look I raise an PR

File: src/pages/Signup.jsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ const SignupScreen = () => {
   const [username, setUsername] = useState("");
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [cpassword, sPassword] = useState("");
```
Comment (narayan954) [inline src/pages/Signup.jsx:25]: cpassword is not readable, and sPassword is worse to read and understand. Try confirmPassword and setConfirmPassword.

File: src/pages/Signup.jsx [hunk 2]
```diff
@@ -36,6 +37,13 @@ const SignupScreen = () => {
     setShowPassword(!showPassword);
   };
 
+  const [showCPassword, setCShowPassword] = useState(false);
```
Comment (narayan954) [inline src/pages/Signup.jsx:40]: try showConfirmPassword and setShowConfirmPassword. Readability is necessary for a clean code base

**Review by narayan954 — CHANGES_REQUESTED** [review]
some minor changes (variable name) required. Also, I forced updated the branch , so try `git pull -f` to update your branch forcefully before making changes, else you'll have to forcefully push your branch using git push -f

**Comment by narayan954** [comment]
closed as completed on another pr

**Pull request closed without merging**